doc/reference/tt_cli/arguments.rst

```diff
@@ -0,0 +1,35 @@
+tt CLI arguments
+================
+
+``tt`` has the following arguments:
+
+..  container:: table
+
+    ..  list-table::
+        :widths: 30 70
```

Comment:
```suggestion
        :widths: 40 60
```
Try these proportions and correct them if necessary. Now the important table contents break in the middle on smaller screens (mine is 13", 1280x580 in Mozilla Firefox):

![image](https://user-images.githubusercontent.com/40490252/177940002-8fe23c89-5af2-45cd-8f26-bfe5666b2725.png)